### PR TITLE
Fix SentTransaction docs

### PR DESCRIPTION
**Pull request by cptartur**
Closes 

## Introduced changes

- This PR fixes the docs generation for fields of SentTransaction subclasses

Before 

image

After

image

##

- [ ] This PR contains breaking changes

### CODE REVIEW

**Review by Solpatium — APPROVED** [review]

File: docs/contract.rst [hunk 1]
```diff
@@ -16,12 +16,15 @@ Contract
...
     :exclude-members: __init__, __new__
-    :members: wait_for_acceptance, wait_for_acceptance_sync
+    :members: wait_for_acceptance, wait_for_acceptance_sync, deployed_contract, hash, status, block_number
+    :member-order: groupwise
 
 .. autoclass:: DeclareResult
     :exclude-members: __init__, __new__
-    :members: deploy, deploy_sync, wait_for_acceptance, wait_for_acceptance_sync
+    :members: deploy, deploy_sync, wait_for_acceptance, wait_for_acceptance_sync, class_hash, compiled_contract, hash, status, block_number
+    :member-order: groupwise
```
Comment (drknzz) [inline docs/contract.rst:30]: Maybe we should consider setting the member-order to `bysource`, as suggested [here](https://github.com/software-mansion/starknet.py/pull/544).
I think having the attribute members just below the class definition is more natural.

Comment (cptartur) [reply docs/contract.rst:30]: I tried bysource and for some reason it was like: properties from class, methods and inherited methods, inherited properties.

So it'd put hash etc.  after wait_for_acceptance but class_hash before wait_for_acceptance (in DeclareResult case)

Comment (drknzz) [reply docs/contract.rst:30]: I'll investigate this

Comment (cptartur) [reply docs/contract.rst:30]: Okay, let me try that

Comment (cptartur) [reply docs/contract.rst:30]: It works except for the DeclareResult.

Code like this

```rst
.. autoclass:: DeclareResult
    :exclude-members: __init__, __new__
    :members: class_hash, compiled_contract, hash, status, block_number, deploy, deploy_sync, wait_for_acceptance, wait_for_acceptance_sync
    :member-order: bysource
```

produces the output like

image

Comment (cptartur) [reply docs/contract.rst:30]: Having it `groupwise` places methods on top, but at least properly separates them from the dataclass fields which I think is better.

**Review by drknzz — APPROVED** [review]
👍

**Review by war-in — APPROVED** [review]

**Pull request merged**